test/presentation-definition/VC_expiration_example.json

```diff
@@ -2,22 +2,25 @@
   "id": "drivers_license_information",
   "name": "Verify Valid License",
   "purpose": "We need you to show that your driver's license will be valid through December of this year.",
-  "schema": [
-    {
-      "uri": "https://yourwatchful.gov/drivers-license-schema.json",
-      "required": true
-    }
-  ],
   "constraints": {
+    "types": [
+      {
+        "path": ["$.credentialSchema.id", "$.vc.credentialSchema.id"],
+        "filter": {
+          "type": "string",
+          "const": "https://yourwatchful.gov/drivers-license-schema.json"
+        }
+      }
+    ],
     "fields": [
       {
         "path": ["$.expirationDate"],
         "filter": {
           "type": "string",
           "format": "date-time",
```

Comment:
notably, the newest version of json schema removes the format property. this may be considered in PEv2

Reply:
Interesting could you send me a link to that? I hadn't heard that format was going away, what is replacing it?